js/views/notificationVw.js

```diff
@@ -19,34 +19,34 @@ module.exports = baseVw.extend({
   },
 
   notificationClick: function() {
-    switch (this.model.get('type')) {
-    case "follow":
-      Backbone.history.navigate('#userPage/'+ this.model.get('guid') + '/store', {trigger: true});
-      break;
-    case "new order":
-      Backbone.history.navigate('#transactions/sales/' + this.model.get('order_id'), {trigger: true});
-      break;
-    case "dispute_open":
-      Backbone.history.navigate('#transactions/purchases/' + this.model.get('order_id'), {trigger: true});
-      break;
-    case "dispute_close":
-      Backbone.history.navigate('#transactions/purchases/' + this.model.get('order_id'), {trigger: true});
-      break;
-    case "refund":
-      Backbone.history.navigate('#transactions/purchases/' + this.model.get('order_id'), {trigger: true});
-      break;
-    case "payment received":
-      Backbone.history.navigate('#transactions/purchases/' + this.model.get('order_id'), {trigger: true});
-      break;
-    case "order confirmation":
-      Backbone.history.navigate('#transactions/purchases/' + this.model.get('order_id'), {trigger: true});
-      break;
-    case "rating received":
-      Backbone.history.navigate('#transactions/sales/' + this.model.get('order_id'), {trigger: true});
-      break;
-    case "ORDER":
-      Backbone.history.navigate('#transactions/sales/' + this.model.get('subject') +'/discussion', {trigger: true});
-      break;
+    switch (this.model.get('type').toLowerCase()) {
+      case "follow":
+        Backbone.history.navigate('#userPage/'+ this.model.get('guid') + '/store', {trigger: true});
+        break;
+      case "new order":
+        Backbone.history.navigate('#transactions/sales/' + this.model.get('order_id'), {trigger: true});
+        break;
+      case "dispute_open":
+        Backbone.history.navigate('#transactions/purchases/' + this.model.get('order_id'), {trigger: true});
+        break;
+      case "dispute_close":
```

Comment:
In the server ticket, I would also request consistency in format beyond case. For example, some types have underscores for spaces (dispute_close) whereas others use actual spaces (new order).